README.md

```diff
@@ -33,42 +33,61 @@ The `bimm` packages requires binary, repeated/clustered outcomes and complete da
 library(bimm)
 ```
 
-We use the hospital dataset in the `bimm` package and split it into training and testing data by patient ID (DID). 
+We use the hospital dataset in the `bimm` package and split it into training and testing data by patient ID (DID) for test_newsubj and testing data for the last observation for each patient for test_newobs. 
```

Comment:
These are very helpful changes! I like the additional documentation for prediction types. Thank you!